app/services/report_importer.rb

```diff
@@ -88,6 +89,10 @@ def report_status
     raise NotImplementedError
   end
 
+  def statuses_for_refresh
+    HostStatus.status_registry
```

Comment:
I wonder, is there a way to infer which status changed from the importer class rather then defaulting to refreshing all of them and having to edit all of the importers manually?